src/MoBi.Presentation/Tasks/SimulationCommitTask.cs

```diff
@@ -24,6 +23,12 @@ public interface ISimulationCommitTask
       /// </summary>
       /// <returns>An executed command</returns>
       IMoBiCommand CommitSimulationChanges(IMoBiSimulation simulationWithChanges);
+
+      /// <summary>
+      ///    Replaces the project default simulation settings with the simulation settings from
+      ///    <paramref name="simulationWithChanges" />
+      /// </summary>
+      ICommand CommitSimulationSettings(IMoBiSimulation simulationWithChanges);
```

Comment:
This one, and the corresponding refresh could be done within CommitSimulationChanges instead. I separated them because it feels a little different. There is no building block in the module explorer to match with. Also, for commit you need to do everything or nothing. So you could just update/commit settings and not everything else.